package.json

```diff
@@ -1,29 +1,29 @@
 {
-  "name": "@socketsupply/socket-api",
+  "name": "@socketsupply/socket",
```

Comment:
We will need to do something with https://github.com/socketsupply/socket/blob/master/npm/packages/%40socketsupply/socket/package.json

Reply:
Actually, since this is private, we can drop the `name` field all together